CP05MOAS-GL376/CP05MOAS-GL376_R00012_ingest.csv

```diff
@@ -1,6 +1,6 @@
 parser,filename_mask,reference_designator,data_source,status,notes
 mi.dataset.driver.moas.gl.engineering.glider_eng_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL376/R00012/merged/cp_*.mrg,CP05MOAS-GL376-00-ENG000000,recovered_host,Available,
-mi.dataset.driver.moas.gl.adcpa.adcpa_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL376/R00012/dvl/*.PD0,CP05MOAS-GL376-01-ADCPAM000,recovered_host,Available,
+#mi.dataset.driver.moas.gl.adcpa.adcpa_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL376/R00012/dvl/*.PD0,CP05MOAS-GL376-01-ADCPAM000,recovered_host,Not Available ,DVL disabled; no data available

```

Comment:
No DVL data available for this deployment. commented out line, changed status and added comment